### PR TITLE
Incorporate changes from #344

### PR DESCRIPTION
**Pull request by KronosTheLate**
I finally made a branch in the correct way, and incorporated the changed made in https://github.com/JuliaControl/ControlSystems.jl/pull/344/. I left out the changes for the linestyles, as I honestly do not know their effect anymore, and the code-base has changed significantly.

Porting 344 was done by copy-pasting and incoperating to the best of my ability. The original changes were however made long ago, so this PR definitively needs some external checking.

### CODE REVIEW

**Comment by KronosTheLate** [comment]
> not modify the section headers?

I changed them for a reason, but fair enough. Done ^_^

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.colorTheme": "Julia (Monokai Vibrant)"
```
Comment (albheim) [inline .vscode/settings.json:2]: Was this really supposed to be added?

Comment (baggepinnen) [reply .vscode/settings.json:2]: oops, no certainly not :)

Comment (KronosTheLate) [reply .vscode/settings.json:2]: I am sorry, no. VSCode keeps adding this file in every folder I open with VSCode, which has made its way into more than one git commit. I will make a PR to remove it.

Comment (KronosTheLate) [reply .vscode/settings.json:2]: Deleted the folder in https://github.com/JuliaControl/ControlSystems.jl/pull/882

**Pull request merged**